pages/main.scss

```diff
@@ -51,6 +52,12 @@ p {
     color: $body-color !important;
   }
 }
+.bg-typography {
```

Comment:
Bootstrap as a concept of bg-inverse, that uses the grey as the background and sets the text colour to be white. Please use that. http://v4-alpha.getbootstrap.com/components/utilities/#contextual-colors-and-backgrounds